README.md

```diff
@@ -33,6 +33,14 @@ See the [usage documentation](docs/en/index.md) for more information.
     composer: `composer require silverstripe/secureassets dev-master`
  2. Run /dev/build?flush=1
 
+## Upgrade Instructions
+As above, this module is no longer needed in the SilverStripe 4.x release line. When you upgrade your project from the 
+SilverStripe 3.x to SilverStripe 4.x, ensure you export the database from your SS3 project and import it 
+into your SS4 project. 
```

Comment:
I think devs would normally upgrade their project, so it'd be the same project. We could suggest that devs backup the database before performing the upgrade, but I don't think this is necessary to mention at a module level. You could leave "no longer needed" but remove the rest.